rs/migration_canister/src/canister_state.rs

```diff
@@ -130,6 +121,25 @@ pub mod events {
         HISTORY.with_borrow(|h| h.keys().collect())
     }
 
+    pub fn num_successes_in_past_24_h() -> u64 {
+        let now = time();
+        let nanos_in_24_h = 24 * 60 * 60 * 1_000_000_000;
+        HISTORY.with_borrow(|h| {
+            let mut count = 0;
+            for event in h.iter_from_prev_key(&Event {
+                time: now.saturating_sub(nanos_in_24_h),
```

Comment:
If we want to use this approach, then we should add a comment to `Event` that time must be the first field declared in the struct - otherwise, the (lexicographic) ordering might be event-first, time-second which is not what we want here.